packages/pn-personafisica-webapp/public/locales/en/recapiti.json

```diff
@@ -1,134 +1,135 @@
-    "disclaimer-message": "This is the primary address that will be used to send you your digital acknowledgement of receipt. By entering it, you will no longer receive registered paper mail.",
+    "pec-added-successfully": "Certified email address correctly associated",
+    "pec-added": "Associated certified email address",
+    "pec-validating": "Certified email address validation in progress",
+    "cancel-pec-validation": "Cancel validation",
+    "remove-pec-title": "Remove certified email address",
+    "remove-pec-message": "Are you sure you want to remove the certified email address {{value}}?",
+    "disclaimer-message": "This is the primary address that will be used to send you acknowledgement of receipts digitally. By entering it, you will no longer receive registered mail.",
     "disclaimer-link": "Find out more",
-    "block-remove-pec-title": "You cannot delete the PEC",
-    "block-remove-pec-message": "In order to do this, you must first delete all the PEC addresses that you indicated under \"Other contact details\".",
+    "block-remove-pec-title": "You cannot delete the certified email address",
+    "block-remove-pec-message": "In order to do this, you must first delete all the certified email addresses you have indicated under \"Other contact details\".",
     "validation-cancel-title": "Do you want to cancel the verification?",
-    "validation-cancel-content": "If you cancel the verification, you will have to enter a new PEC and start the process again.",
-    "validation-in-progress": "Verifying",
-    "validation-progress-title": "Thank you, now it's our turn",
-    "validation-progress-content": "We are verifying that the address you entered is a valid PEC for receiving notifications from the Public Administration. You will receive the outcome of the verification via email."
+    "validation-cancel-content": "If you cancel the verification, you will need to enter a new certified email address and restart the process.",
+    "validation-in-progress": "Verification in progress",
+    "validation-progress-title": "Thank you!",
+    "validation-progress-content": "We are verifying that the certified email address is valid. You will receive an email with the result within 24 hours."
   },
   "io-contact": {
     "title": "",
     "subtitle": "IO app",
-    "description": "Activate the \"Digital notifications\" service: when there is a notification for you, we send you a message in the app, where you can read it and pay any charges. Here you will also receive any important communications.",
-    "disclaimer-message": "If you do not have a PEC, read the notification immediately: you will not receive the registered mail and you will save time and money.",
+    "description": "Activate the \"Digital Notifications\" service: when there is a notification for you, we send you a message in the app, where you can read it and pay any expenses. Here, you will also receive any important communications.",
```

Comment:
@AliceAzzolini non ti taggo in ogni punto, ma anche il nome del servizio "Notifiche digitali" ha senso che venga tradotto?